podstawowe_operacje/Installment/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿var price = (decimal)-1;
```

Comment:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/floating-point-numeric-types
```suggestion
﻿var price = -1m;
```

Reply:
[c26dc60](https://github.com/broniuu/Wst-pDoProgramowania/pull/1/commits/c26dc608bd9a89fb30ff8675d876c3bcc134ac77)

Reply:
OK